File: Easy/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        check_val = {}
        
        for i in range(len(nums)):
            second_num = target - nums[i]
            
            if(second_num in check_val.keys()):
                second_ind = nums.index(second_num)
                if(i != second_ind): return sorted([i, second_ind])
                
            check_val.update({nums[i]: i})
            
        return check_val

File: Easy/Swift/two_sum.swift
class Solution {

    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var check_val = [Int: Int]()

        for (i, n) in nums.enumerated() {

            let second_num = target - n
            
            if let second_ind = check_val[second_num] {
                return [i, second_ind]
            }
            
            check_val[n] = i
        }

        return []
    }
}
